src/wrapper.rs

```diff
@@ -0,0 +1,235 @@
+use pyo3::{
+    intern,
+    pyfunction,
+    types::{
+        PyModule,
+        PyString,
+        PyTuple,
+    },
+    IntoPy,
+    Py,
+    PyAny,
+    PyResult,
+    Python,
+};
+
+use crate::py_err;
+
+/// Create a path name to use for pipe forwarding.
+/// This will be a file path in the temp directory on unix-like systems,
+/// or a named pipe on Windows. Files will be removed at program exit.
+#[pyfunction]
+pub fn pipe_name(py: Python) -> PyResult<Py<PyAny>> {
+    call_code(
```

Comment:
:eval_intensifies:

Reply:
Yeah, gives me a funny feeling, but it's all CI tested, and very similar to the `trailer.js` on the NodeJS side.